File: Palace/OPDS/TPPOPDSAttribute.m
BOOL TPPOPDSAttributeKeyStringIsActiveFacet(NSString *const string)
{
  return string != nil && [string rangeOfString:@"activeFacet"
                       options:NSCaseInsensitiveSearch].location != NSNotFound;

}

BOOL TPPOPDSAttributeKeyStringIsFacetGroup(NSString *const string)
{
  return string != nil && [string rangeOfString:@"facetGroup"
                       options:NSCaseInsensitiveSearch].location != NSNotFound;

}

BOOL TPPOPDSAttributeKeyStringIsFacetGroupType(NSString *const string)
{
  return string != nil && [string rangeOfString:@"facetGroupType"
                       options:NSCaseInsensitiveSearch].location != NSNotFound;
}

File: Palace/OPDS/TPPOPDSAttribute.swift
import Foundation

func TPPOPDSAttributeKeyStringIsActiveFacet(_ string: String?) -> Bool {
  guard let string = string else { return false }
  return string.range(of: "activeFacet", options: .caseInsensitive) != nil
}

func TPPOPDSAttributeKeyStringIsFacetGroup(_ string: String?) -> Bool {
  guard let string = string else { return false }
  return string.range(of: "facetGroup", options: .caseInsensitive) != nil
}

func TPPOPDSAttributeKeyStringIsFacetGroupType(_ string: String?) -> Bool {
  guard let string = string else { return false }
  return string.range(of: "facetGroupType", options: .caseInsensitive) != nil
}
